scripts/start_tendermint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
```

Comment:
The name of the script is misleading. It starts more than just tendermint, it starts nearmint app, too. Rename to run_nearmint.sh (to also be consistent with `ops`)?

Reply:
done